emodelrunner/protocols/thalamus_protocols.py

```diff
@@ -514,6 +514,7 @@ def generate_current(self, threshold_current=None, holding_current=None, dt=0.1)
         total_duration = self.rin_protocol.step_stimulus.total_duration
 
         t = np.arange(0.0, total_duration, dt)
+        t = np.append(t, t[0])  # voltage shape is always total_duration*dt+1
```

Comment:
Why appending t[0] at the end of the array? Wouldn't it make more sense to append t[-1] at the end?

Reply:
Yes, right. Let me fix it